src/ngraph_builder1.h

```diff
@@ -0,0 +1,138 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+#ifndef NGRAPH_TF_BRIDGE_BUILDER1_H_
+#define NGRAPH_TF_BRIDGE_BUILDER1_H_
+
+#include <ostream>
+#include <vector>
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph_log.h"
+#include "ngraph_mark_for_clustering.h"
+
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/tensor_shape.h"
+#include "tensorflow/core/graph/algorithm.h"
+
+using namespace std;
+namespace ng = ngraph;
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+// TODO (sarkars): make sure all comments from old builder are copied correctly.
+
+// The purpose of Builder is to accept a TF graph and convert it to a
+// equivalent ngraph fucntion. To that end, it acts as an interface to
+// a library of 'TranslateOps' functions. These functions are defined
+// in ngraph_translateops.h. Builder just goes through each TF op and
+// calls the appropriate 'TranslatorFn'. It does not implement the per-op
+// translation, the heavy-lifting being done by ngraph_translateops.h;
+// It merely does some bookkeeping, such as process the nodes in correct
+// order, keep track of and retrieve parent nGraph nodes etc.
+
+class Builder1 {
+  using VectNg = std::vector<shared_ptr<ng::Node>>;
```

Comment:
Nitpick: I'm not a fan of this particular instance of `using`. (It obscures the fact that the elements are pointers, and that they are pointers to nodes, for one thing.)

Reply:
How about VectPtrNg?
This std::vector<shared_ptr<ng::Node>> gets used a lot, so wanted some typedef

Reply:
removed the typedef